contents/docs/webhooks/index.md

```diff
@@ -26,13 +26,21 @@ The parts in square brackets `[]` are called _message tokens_. You can use token
 - `[event.event]`: Same as `[event]` except not formatted as a link.
 - `[event.uuid]`: ID of the event. Always in UUID format.
 - `[event.distinct_id]`: Person distinct ID associated with the event.
-- `[event.properties.property_name]`: Value of property `property_name` – e.g., `[event.properties.$os]`, `[event.properties.amountUSD]`, or `[event.properties.object.nested_prop]`.
+- `[event.properties.<property_name>]`: Value of property `<property_name>` – e.g., `[event.properties.$os]`, `[event.properties.amountUSD]`, or `[event.properties.object.nested_prop]`.
```

Comment:
?
```suggestion
- `[event.properties.<property_name>]`: Value of property `<property_name>` – e.g., `[event.properties.$os]`, `[event.properties.amountUSD]`, or `[event.properties.object.<nested_prop>]`.
```

Reply:
That's an example. I don't think it needs a template.